tests/sflow_test.go

```diff
@@ -161,6 +160,47 @@ func TestSFlowAgent(t *testing.T) {
 
 		/* FIXME (nplanel) remove this Sleep when agent.FlushFlowTable() exist */
 		time.Sleep(2 * time.Second)
-		ts.Validate(t, &trace)
+		pcapTraceValidate(t, ts.GetFlows(), &trace)
+	}
+}
+
+func TestSFlowProbePath(t *testing.T) {
+	hostname, err := os.Hostname()
+	if err != nil {
+		t.Fatal(err.Error())
+	}
+
+	ts := NewTestStorage()
+
+	agent, analyzer := helper.StartAgentAndAnalyzerWithConfig(t, confAgentAnalyzer, ts)
+	defer agent.Stop()
+	defer analyzer.Stop()
+
+	time.Sleep(1 * time.Second)
+	setupCmds := []helper.Cmd{
+		{"ovs-vsctl add-br br-sflow", true},
+		{"ovs-vsctl add-port br-sflow sflow-intf1 -- set interface sflow-intf1 type=internal", true},
+		{"ip address add 10.0.0.1/24 dev sflow-intf1", true},
```

Comment:
May change the subnet to a much random one, like 10.254.178.1/24   or something else...
Too many setup have already a subnet set to 10.0.0.1/xx